docs/001.tm

```diff
@@ -19,7 +20,21 @@
 
   <section|How to test it?>
 
+  <\shell-code>
```

Comment:
only work for macOS and GNU/Linux

For Windows, we have not support it yet. It should be resolved in issue 005.tm.